src/ui/tests/e2e/orders/statusChange.spec.ts

```diff
@@ -0,0 +1,191 @@
+import { ALERTS, TAGS } from 'data';
+import { DELIVERY_CONDITIONS, ORDER_STATUSES } from 'data/orders';
+import { test, expect } from 'fixtures';
+import { IOrderFromResponse } from 'types';
+import { generateValidDeliveryDate } from 'utils';
+
+test.describe('[E2E] [UI] [Orders] [Status Change]', () => {
+  let token = '';
+  let order: IOrderFromResponse;
+  const managerData = { id: '68196484d006ba3d4760a076', name: 'Barys Kotau' };
+
+  test.beforeEach(async ({ signInApiService }) => {
+    token = await signInApiService.getAuthToken();
+  });
+  test.afterEach(async ({ ordersApiService }) => {
+    await ordersApiService.clear(token);
+  });
+
+  test(
+    'Should cancel order on Draft',
+    { tag: ['@001_O_SC_E2E', TAGS.E2E] },
+    async ({ ordersApiService, orderDetailsPage }) => {
+      order = await ordersApiService.createDraft(token);
+      order = await ordersApiService.assignManager(order._id, managerData.id, token);
+      await orderDetailsPage.openPage('ORDER_DETAILS', order._id);
+      await orderDetailsPage.waitForOpened();
+      await orderDetailsPage.clickCancel();
+      await orderDetailsPage.cancelModal.submit();
+      await orderDetailsPage.waitForNotification(ALERTS.ORDER_CANCELED);
+      const status = (await orderDetailsPage.getOrderValues()).status;
+      expect(status).toBe(ORDER_STATUSES.CANCELED);
```

Comment:
Во все экспекты вторым параметром кидай описание пороверки, например

```suggestion
      expect(status, "Verify order status is In Process").toBe(ORDER_STATUSES.CANCELED);
```